src/mn_setup.c

```diff
@@ -1882,7 +1881,7 @@ static setup_menu_t stat_settings2[] = {
      .strings_id = str_show_widgets, .action = UpdateStatsFormatItem},
 
     {"Show Level Time", S_CHOICE, H_X, M_SPC, {"hud_level_time"},
-     .strings_id = str_show_widgets, .action = UpdateUseButtonItem},
```

Comment:
I think if we allow to enable "Use-Button Timer" independent of "Show Level Time", we should move it back up into the "Widget Types" section.

Reply:
Sure. Something like "Show Use-Button Timer" would be consistent with the others.

How should this interact with the stats toggle key binding?

Reply:
> Sure. Something like "Show Use-Button Timer" would be consistent with the others.

Yes, I will move it there.  

> How should this interact with the stats toggle key binding?

Not sure. Strictly, it's not one of the level stats widgets that are meant by this toggle. I'd leave it as it is. What do you think?

Reply:
@MrAlaux you suggested to uncouple the use-button timer from the level time widget. What do you think?

Reply:
> I'd leave it as it is. What do you think?

That sounds fine to me.

Reply:
> @MrAlaux you suggested to uncouple the use-button timer from the level time widget. What do you think?

Not quite _uncouple_, just make it independent but still existing within the same widget. Anyways, I agree on that it shouldn't be affected by the toggle key.